### ISSUE TITLE
Week 2 Discussion

### ISSUE THREAD

**Issue by acrook** [stefanbund/bc124#1]
After studying the user interface to Adobe Acrobat in week one, how simple do you rate the user experience?

I find that Adobe Acrobat is the most user friendly PDF program, as opposed to Nitro and Bluebeam. .  I tend to have compatibility issues with the latter, which disrupts workflow. In all, it's simple to use, and simple to teach.

**Comment by stefanbund** [comment]
I tend to agree. I suppose that the best way to evaluate the efficiency of the interface is to get heavy users to reflect upon it after much use. However, the path to solutions are easy to remember and build into your work routine.

From Stefan

**Comment by brittanyharter** [comment]
I find the user interface of Adobe Acrobat very simple and easy to use. Even before reading any of the tutorials and just looking at the options on the tools page, I had a pretty good idea on how to use them/what to use them for. Although, I'm sure I still have a lot to learn!

**Comment by stefanbund** [comment]
Brittany, during the first week I simply wanted students to comprehend the breath of the program while  investigating the user interface. This acts as a preview of all the skills you will learn during the course.

From Stefan

**Comment by heather894** [comment]
I think the interface is very simple and easy to understand. As a long time Adobe user I don't think that is/was always the case with their software. Recent versions of acrobat seem much more user friendly to me.

**Comment by stefanbund** [comment]
I think you guys are pretty lucky to be learning that DC interface. It is really appealing and has a lot more open oxygen in the experience. I really enjoy it. Glad you're in my class and learning new stuff

From Stefan

**Comment by pristt** [comment]
Adobe Pro DC instructions are quite simple to follow. The instructions videos in the course are a little different from the tools of Pro DC. It took me some time to navigate between the different applications.

**Comment by yoviedo** [comment]
I find Adobe Pro extremely user friendly. Most of the commands are pretty much self-explanatory. I don't find myself having to go through different options to get the results or tools I want to use. I think this is a great application and used by most if not all organizations.
